### PR TITLE
EPMLSTRCMW-177 Add api collections from postman

### PR DESCRIPTION
**Pull request by GrigoriiBerezin**

### CODE REVIEW

File: postman/Authorization.postman_collection.json [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+					"    var headers = pm.response.headers;",
+					"    pm.environment.set(\"auth_token\", headers.get(\"Access-Token\"));",
+					"    pm.environment.set(\"refresh_token\", headers.get(\"Refresh-Token\"));",
+					"}"
+				]
+			}
+		}
+	],
+	"protocolProfileBehavior": {}
+}
```
Comment (Dmitrii-Shevchenko) [inline postman/Authorization.postman_collection.json:229]: Turn on in idea settings ```Ensure an empty line at the end of a file on Save``` to get rid of these red warnings

**Review by Dmitrii-Shevchenko — APPROVED** [review]
Great work!

File: postman/Authorization.postman_collection.json (hunk 1 above)
Comment (GrigoriiBerezin) [reply postman/Authorization.postman_collection.json:229]: Thanks for noticing that, Fixed

**Review by IaroslavTavchenkov — APPROVED** [review]
Thanks a lot for your work! :)

**Review by iliaonishchenko — APPROVED** [review]
Nice!

**Review by myazinn — CHANGES_REQUESTED** [review]
Great job! Such things are really cool. It is really uncomfortable for me to request changes :)
There are places where you could have automated things a little bit more. For example, take a look at how `update password` URL looks
image
I though it will look like this
`{{base_url}}/users`
Because we know how this API looks. We don't now the sever URL, but the specific API part is known. Can you please fix this and other places too? I assume this will not be a problem.

**Comment by GrigoriiBerezin** [comment]
@myazinn Seems legit for me, changed it. Also noticed that environment didn't copy values, so fixed it too

**Review by myazinn — APPROVED** [review]
Looks great to me

**Pull request merged**